### DISCUSSION TITLE
$name.about.tsx does not render when it should

### DISCUSSION THREAD

**Discussion by olyamrshn** [TanStack/router#2789] [Q&A] [answered]
I have [this repo](https://github.com/olyamrshn/zuleika-jewelry) and these routes inside:

-  [$name.tsx](https://github.com/olyamrshn/zuleika-jewelry/blob/main/app/routes/collection/%24name.tsx)
-  [$name.about.tsx](https://github.com/olyamrshn/zuleika-jewelry/blob/main/app/routes/collection/%24name.about.tsx)

I thought if I visit http://localhost:3000/collection/eden/about, it would render `$name.about.tsx` but it renders `$name.tsx` still

Not sure what I am doing wrong.

**Comment by schiller-manuel** [comment, marked as answer]
`$name` needs to render an `<Outlet>` so that the child route can be rendered there

**Reply by olyamrshn** [reply]
thank you, it works! <З
